app/build.gradle

```diff
@@ -166,6 +166,7 @@ dependencies {
     implementation "net.sf.supercsv:super-csv:$versions.super_csv"
     implementation 'com.android.support:customtabs:28.0.0'
     implementation 'com.jaredrummler:android-device-names:2.0.0'
+    implementation 'net.danlew:android.joda:2.10.3'
```

Comment:
Joda-time is not used on Android much, kind of deprecated. New apps use java.time.* API, backported by this library: https://github.com/JakeWharton/ThreeTenABP Is it easy to swap?

Reply:
Oh. I've tried to use `java.time` directly but it's API 26+. I'll try but if it's a true backport, it should be pretty easy to do that.

Reply:
Updated. It was easy indeed.